### DISCUSSION TITLE
Does the thorium construction project seem to have stalled?

### DISCUSSION THREAD

**Discussion by whxuezhe** [Alex313031/Thorium-Win-AVX2#58] [General]
It is known that M109 was released on October 18, and it has been nearly two months now. Are you going to follow 111.0 directly beyond 110.0? 
Or is it considering the rights and interests of the win7 user base and formally issuing the enforcement ban before considering the next project promotion? 
Related to the GPU crash, I think we should optimize and update the packets of the built-in decoding library, as well as the compatibility testing of M3U8 HLS online streaming media in different formats.

**Comment by gz83** [comment]
> It is known that M109 was released on October 18, and it has been nearly two months now. Are you going to follow 111.0 directly beyond 110.0? Or is it considering the rights and interests of the win7 user base and formally issuing the enforcement ban before considering the next project promotion? Related to the GPU crash, I think we should optimize and update the packets of the built-in decoding library, as well as the compatibility testing of M3U8 HLS online streaming media in different formats.

@whxuezhe Hello, the project will be updated in the near future. Please pay attention to the dynamics in the repo. Recently, we plan to make a specific version for the win7/8/8.1 system.
